rixin/updated teams graphics and set up styling for new teams page

rixinhaha opened the pull request:
* added rotated and normal team header graphics
* handled conditional display of rotated/normal headers
* modified grid layout for teams page
* updated styling for h1 elements
* cleaned up on unused styles for teams.css

rixinhaha committed:
rixin/updated teams graphics and set up styling for new teams page

robertnperalta commented on views/team.hbs:
```diff
@@ -9,42 +10,90 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
     <!-- Font -->
     <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:300&display=swap" rel="stylesheet" />
```
We aren't using Roboto Condensed, so remove this `<link>`. We are, however using Roboto, so be sure to add it to the `<link>` with Source Code Pro.

rixinhaha replied:
not super sure how to do this
I went to google fonts and it seems like i can't select the entire Roboto font, I can only add the specifics styles.
` <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet"> `

robertnperalta replied:
Check the [website notes](https://docs.google.com/document/d/1nckfyQzNbhywWvx4oU0LGE16ppUeYC5eiBEEcFlKsf4/edit?usp=sharing).

That said, you do technically only need to include the weights you need and even then, the CSS-generated bold is barely different than the Google fonts version. `<link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">` defaults to 400 weight, which is generally enough if you're just using normal and bold text.

rixinhaha replied:
is this line fine?
`<link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>`

rixinhaha replied:
fixed this and added additional margin

robertnperalta replied:
Don't use that line for a couple reasons: it uses the old Google fonts api (notice it has `css` not `css2`), and it's best to combine the link with the Source Code Pro link so the webpage only needs to do one request to load, instead of two

robertnperalta commented on public/css/team.css:
```diff
@@ -3,12 +3,26 @@ body {
+
 .team-container {
 	text-align: center;
 	font-family: "Roboto Condensed";
```
Remove this line for clarity, even though it's being overwritten everywhere.

robertnperalta replied:
It displayed the lines I commented on weirdly; I'm talking about the `font-family` line.

rixinhaha replied:
Fixed

robertnperalta commented on public/css/team.css:
```diff
@@ -32,38 +46,19 @@ p {
 .Description {
-	color: #2047a4;
-	font-family: "Roboto Condensed";
+	color: #000000;
```
We're using `#2b2b2b` as our text color, not solid black.

rixinhaha replied:
Fixed

robertnperalta commented on views/team.hbs:
```diff
@@ -9,42 +10,90 @@
+            </div>
+            {{else}}
+            {{#if_even @index}}
+            {{#if this.teamPath}}<img src="{{this.teamPath}}" alt="LOGO" class="header">{{/if}}
+            <div class='container-fluid section'>
+                <div class='individual-team__container reversed'>
+                    <div class="header-rotated-wrapper">{{#if this.teamPath}}<img src="{{this.teamPathRotated}}"
+                            alt="LOGO" class="header-rotated ml30">{{/if}}
```
Change the `alt` attributes to match what the header actually says. You could do it something like this:
```suggestion
            {{#if this.teamPath}}<img src="{{this.teamPath}}" alt="{{this.departmentName}}" class="header">{{/if}}
            <div class='container-fluid section'>
                <div class='individual-team__container reversed'>
                    <div class="header-rotated-wrapper">{{#if this.teamPath}}<img src="{{this.teamPathRotated}}"
                            alt="{{this.departmentName}}" class="header-rotated ml30">{{/if}}
```
You can also hard-code it for the directors section.

Same goes for the `alt` attributes lower down in the code.

rixinhaha replied:
i will use this.departmentName is that okay?

robertnperalta replied:
Absolutely

robertnperalta reviewed (changes requested):
On top of the commented sections of code, add a little more space between each `.individual-team__container`. It can feel crowded on wide screens where the vertical logo is longer than the photos; the logo ends up getting too close to the next committee.

Other than these small changes though, it looks really good!

rixinhaha committed:
rixin/teams-page
*added additional margin between each teams-container
*changed font-color
*added meaningful alt text for header images
*removed Roboto-Condensed

robertnperalta committed:
Clean up font styling

robertnperalta committed:
Run Prettier

robertnperalta reviewed (approved):
Lgtm!
